packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.Input/src/InputTypes/InputModelProperty.cs

```diff
@@ -35,6 +35,7 @@ public InputModelProperty(
         public bool IsDiscriminator { get; internal set; }
         public InputSerializationOptions? SerializationOptions { get; internal set; }
         public bool IsHttpMetadata { get; internal set; }
+        public bool IsFlattened { get; internal set; }
```

Comment:
It doesn't make sense to include IsFlattened here if we are not actually honoring it in MTG. In general, there could be other decorators that mgmt/dpg want to support, but we wouldn't add properties for each of those, right? We should either implement flattening in MTG, or just add a dictionary of additional metadata.